packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -23,7 +23,9 @@
   "categories": ["Languages"],
   "dependencies": {
     "lwc-language-server": "0.14.1",
-    "vscode-languageclient": "3.5.0"
+    "vscode-languageclient": "3.5.0",
+    "eslint": "4.15.0",
+    "eslint-plugin-lwc": "0.2.0"
```

Comment:
latest version is 0.2.4, but we could upgrade to newer once we tune the enabled eslint rules for the pilot

Reply:
Sounds good.